### DISCUSSION TITLE
Remaining Tasks & Questions for 1.0

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#830] [Announcements]
After skimming through the [1.0 milestone tasks](https://github.com/idom-team/idom/milestone/1), the largest remaining tasks for 1.0 are:

- https://github.com/idom-team/idom/issues/574
- https://github.com/idom-team/idom/issues/755
- and finishing the documentation

An additional question that has remained unanswered is whether 1.0 should be released under the IDOM name at all. At the moment we own `reactive` on PyPI, a [reactive-python](https://github.com/reactive-python/reactive) org on GitHub, and the https://reactive-python.com domain. So if we wanted we'd be set to make that change. As we've talked about before, the name may not be ideal from a marketing perspective, but I think there may be ways of branding the project that could work around this. For example, referring to the project as "reactive.py" instead of just "Reactive" or "Reactive Python". Making this change would itself be a significant undertaking, but one that I think is worth doing.

**Comment by Archmonger** [comment]
I think we go forward with the rename, it's far better than what we currently have.

**Reply by Archmonger** [reply]
Might be worth pinging the ReactPy developer once more. He hasn't ever put any development into his project so it'd be nice to own that name.

**Reply by rmorshea** [reply]
I pinged him again a week or two ago and no response unfortunately.

**Reply by Archmonger** [reply]
We should ping him a week before out official rename and see if he's willing to contact us back.

**Comment by rmorshea** [comment]
@Archmonger, what do you think about the name "Reacton"? I'm in communication with the owner of [that package](https://github.com/widgetti/reacton) and they'd be willing to merge efforts. For example, the present work to create a "React, but for IpyWIdgets" would go under a `reacton.ipywidgets` namespace package.

**Reply by Archmonger** [reply]
I think that's a downgrade from `reactive`.

For non-compound names (single words), the main question to ask is "What does the word ____ mean?"

In terms of `reacton` it's not really a word, so it falls into the same lack of easy memorability pitfall as `idom`.

**Comment by tylerhjones** [comment]
I think a rename could improve search results. As a new user, right now most results are 'idiom', or 'idiomatic' related and not the IDOM framework.

**Reply by Archmonger** [reply]
ReactPy was my first proposition, but someone has already name parked it and is not willing to provide it to us.

Right now it looks like we are going to rename to Reactive-Python
